src/scenario.jl

```diff
@@ -939,25 +944,39 @@ function run_model(
         has_seed_locs::Bool = !isempty(selected_seed_ranks)
 
         # Apply seeding (assumed to occur after spawning)
-        if seed_decision_years[tstep] && has_seed_locs
+        if has_seed_locs  # seed_decision_years[tstep] &&
```

Comment:
Either we maintain the seed decision year check
```suggestion
        if has_seed_locs && seed_decision_years[tstep]
```

or the variable `selected_seed_ranks` must be assigned to be empty at the beginning of every timestep, otherwise ADRIA will attempt to carry out seeding when its not a seed year and will use the previous seeding years locations.

Reply:
Yes, that's the idea. See my comment.

Deployments should now occur every year, but only the change in locations vary every n years where n could be 0 (select locations once and keep deploying in the same locations), 1 (change every year), etc.

Reply:
Interesting that you get an error, I haven't come across it just yet.